feat(store-sync): add support for watching pending logs (#3287)
latticexyz/mud: .changeset/mean-squids-buy.md

---
"@latticexyz/store-sync": patch
---

> Add experimental support for syncing state from pending logs to the sync stack.

Reviewer: ```suggestion
Added experimental support for syncing state from pending logs.
```
